ddtrace/contrib/aiohttp/__init__.py

```diff
@@ -1,22 +1,42 @@
 """
-The ``aiohttp`` integration traces all requests received by defined routes
-and handlers. External modules for database calls and templates rendering
-are not automatically instrumented, so you must use the ``patch()`` function::
+The ``aiohttp`` integration traces all requests defined in the application handlers.
```

Comment:
Now we have only one section; fixed here: https://github.com/DataDog/dd-trace-py/pull/191/files#diff-e210ccb037c02543f7607dd5fa9ef490R9